frontend/src/components/design-system/containers/PillComponent.vue

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+export enum ColorPill {
+  Light,
+  Dark,
+}
+</script>
+
+<script setup lang="ts">
+export interface Props {
+  icon: boolean;
+  color?: ColorPill;
+  title: string;
```

Comment:
Title byl myšlen jako ten obsah? To by mělo být ve slotu, tj. vyhodit z props.

Reply:
To mi nedošlo. Je to pryč.